### DISCUSSION TITLE
Should the Tomb of Tihocan "secret sound" be fixed to how it was intended by devs?

### DISCUSSION THREAD

**Discussion by Rapora9** [LostArtefacts/TRX#432] [Ideas]
I know the secret sound playing in Tomb of Tihocan is fixed already, but it's not working as was probably designed by Core. There is an old issue #70 for it that was marked invalid already, but I still wanted to raise a discussion for this. So here's my case:

1. Using the Gold Key activates a flip map to raise the "chain of blocks", and also flipeffect 15.
![Näyttökuva (281)](https://user-images.githubusercontent.com/97730054/153311161-a59ea61d-798e-40c1-bbdf-3297fc698345.png)

2. In (http://xproger.info/projects/OpenLara/trs.html#flipeffects) flipeffect 15 is called "CHAINBLOCK_FX".
![Näyttökuva (282)](https://user-images.githubusercontent.com/97730054/153311854-4bf25b69-f7ff-467c-8746-23f6df1b06bf.png)

3. According to the same site, this flipeffect "Plays global sounds with ID 173 and ID 33 with predefined interval. Used in TR1, Tomb of Tihocan."
![Näyttökuva (283)](https://user-images.githubusercontent.com/97730054/153311998-0f38d86a-7e84-40d4-b71a-005eae780ecf.png)

4. When looking at the sound IDs with SoundTool of Tomb Editor, sound ID 33 is "Lara splash".
![Näyttökuva (284)](https://user-images.githubusercontent.com/97730054/153312095-200a7f37-018b-4b34-a06f-6a0aa844a344.png)

5. Sound ID 173 is "secret" which is why the secret sound is played. Sound ID 172 however is called CHAINBLOCK_FX and the sound file "chain_fx.wav". It looks like a typo happened and the wrong sound got added to the flipeffect 15.
![Näyttökuva (285)](https://user-images.githubusercontent.com/97730054/153312355-aee4545a-d740-42fe-968d-15b82b4a654c.png)
 
6. This is how activating the flipeffect sounds like "normally" = with the secret sound.
https://user-images.githubusercontent.com/97730054/153312520-7fda05b9-ea84-4e28-8c36-2c2762533904.mp4

7. And this is how it sounds like when I replace secret.wav with chain_fx.wav. Notice how the "predefined interval" matches. The idea behind this sound combination (and this is my speculation) is that a chain or something raises the blocks (chain sound) and then the blocks break the surface of the water (lara splash).
https://user-images.githubusercontent.com/97730054/153312729-c4c39b5b-04cf-47ec-a9fd-7150c61de718.mp4

8. This is how it would sound like in the level itself (video is edited):
https://user-images.githubusercontent.com/97730054/153313806-badd33ae-34b0-4655-b3ff-5b6ebaacf81f.mp4

** **

So, what do you think? Is the current fix (only playing the splash sound after using the key) good? Or should this "original intention" be restored? In my opinion, if it's not too much work, it would be a nice gesture of honour towards Core to play the correct sounds (chain_fx and splash) here. Talking directly to @rr- , you said back then in #70 that "It's very metallic whereas the blocks are released and should make a splash." The flipeffect is supposed to play both, the metallic chain sound and then the splash after that :)

**Comment by rr-** [comment]
>The flipeffect is supposed to play both, the metallic chain sound and then the splash after that :)

That makes sense. I'm OK with going through with this.

**Comment by walkawayy** [comment]
Wow nice detective work. makes sense!

**Comment by Richard-L** [comment]
I love this too. Well done Rapora.

The only thing that's odd is how after the key entry in your last video, the camera moves back directly behind Lara, and you can see the block already being there before the splash sound. For some reason I don't remember this camera position 🤔 Will double-check on PS1 later.

**Comment by Rapora9** [comment]
Just adding some more information why the chain sound (probably) is there. At the start of the level there's a similar block on the ground. It rises with the water, indicating that the blocks float. Yet those multiple blocks are not floating even when there's water there already. My theory is that they're kept at the bottom of the water by chains, that then are released. There's no texture to show a hole or anything below them though which I find odd.

![Nayttokuva_291](https://user-images.githubusercontent.com/97730054/153411752-f386ddfb-498b-47bf-8d38-e453720b2871.png)

What comes to the camera, I was speculating whether its FOV or other properties are different on tomb1main/pc and ps1.

I'm glad to hear this change is accepted :)

**Comment by rr-** [comment]
I believe this can be closed, no? @Rapora9

**Reply by Rapora9** [reply]
Yes.
